buildspec-release.yml

```diff
@@ -33,6 +33,15 @@ phases:
       - docker build -t sagemaker-training-toolkit-test:dummy -f dummy/Dockerfile .
       - rm dummy/sagemaker_training.tar.gz
 
+      # build tensorflow container
+      - cd ../..
+      - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin 763104351884.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
```

Comment:
Thanks for the PR. A minor change request:
Please use `us-west-2` instead of `AWS_DEFAULT_REGION` as we are doing ecr auth in line 25 in region us-west-2.